develop/reference_ddl.rst

```diff
@@ -246,11 +246,15 @@ Foreign keys may be created in these situations:
 
 * between two local (non-distributed) tables,
 * between two reference tables,
+* between reference tables and local tables (by default enabled, via :ref:`enable_local_ref_fkeys`),
 * between two :ref:`colocated <colocation>` distributed tables when the key includes the distribution column, or
 * as a distributed table referencing a :ref:`reference table <reference_tables>`
 
 Foreign keys from reference tables to distributed tables are not supported.
 
+For the foreign keys between reference tables and local tables, any `referential action <https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-FK>`_ is allowed for foreign keys from local tables to reference tables.
+However, Citus only supports `RESTRICT` action or no action for foreign keys from reference tables to local tables.
```

Comment:
I think mentioning not support for `ON DELETE/UPDATE CASCADE` could be more useful as they are much more commonly known by the users (as opposed to the meaning of RESTIRCT/NO ACTION`